app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,9 @@
+class LikesController < ApplicationController
+  def create
+    @like = Like.new
+    @like.author = current_user
+    @like.post = Post.find(params[:post_id])
+    @like.save!
+    redirect_to user_post_path(user_id: params[:user_id], id: params[:post_id])
+  end
```

Comment:
- [ ] The user is supposed to be able to add a comment and a like, however what we have here is an error when the user tries to click on each single post. Kindly fix this issue

![image](https://user-images.githubusercontent.com/81932478/216797091-36437f9d-8985-4de4-aa09-31a6ff463075.png)

Result After Clicking on Each Post
![image](https://user-images.githubusercontent.com/81932478/216797118-2248e7a4-2c0c-49eb-bc2c-2026ab9ff40c.png)

Reply:
Needs attention!

Reply:
Sir I updated these files
My files do not look like this right now, and everything is working,

Reply:
> Needs attention!

Could you please check my latest commit?
I entirely updated this file

Thank you

Reply:
well resolved! ✔️

Reply:
> well resolved! ✔️

So could you please review this project?

Thank you

Reply:
Done!
Pls, however, do not introduce new changes when your project is under review!

Cheers!